From ce2699a79714c9c84ab4a58f1650005d812d3ea4 Mon Sep 17 00:00:00 2001
From: Jason Wessel <jason.wessel@windriver.com>
Date: Thu, 21 May 2020 12:40:30 -0700
Subject: [PATCH] vc4-fkms-v3d-overlay.dts: Add hdmi

Without the hdmi entry in the overlay it is impossible to get the text
console graphics to initialize in u-boot.

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>

Upstream-Status: Inappropriate, for temporary use

Signed-off-by: He Zhe <zhe.he@windriver.com>
---
 arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
index d9af97c8414f..808ef4121ac2 100644
--- a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
@@ -71,6 +71,13 @@
 		};
 	};
 
+	fragment@9 {
+		target = <&hdmi>;
+		__overlay__  {
+			status = "okay";
+		};
+	};
+
 	__overrides__ {
 		cma-256 = <0>,"+0-1-2-3-4";
 		cma-192 = <0>,"-0+1-2-3-4";
-- 
2.23.0

